### DISCUSSION TITLE
Getting Load Config error for the DNSd module

### DISCUSSION THREAD

**Discussion by vivekshrivastava1516** [magma/magma#15343] [Q&A]
magma@magmahealth.service: Failed with result 'exit-code'.
vagrant@magma-dev:~/magma/lte/gateway$ sudo tail  /var/log/syslog
Nov 27 08:29:30 magma-dev magmad[1019599]:   File "/home/vagrant/magma/orc8r/gateway/python/magma/magmad/main.py", line 325, in <module>
Nov 27 08:29:30 magma-dev magmad[1019599]:     main()
Nov 27 08:29:30 magma-dev magmad[1019599]:   File "/home/vagrant/magma/orc8r/gateway/python/magma/magmad/main.py", line 132, in main
Nov 27 08:29:30 magma-dev magmad[1019599]:     CONFIG_STREAM_NAME: ConfigManager(
Nov 27 08:29:30 magma-dev magmad[1019599]:   File "/home/vagrant/magma/orc8r/gateway/python/magma/magmad/config_manager.py", line 76, in __init__
Nov 27 08:29:30 magma-dev magmad[1019599]:     self._mconfig = self._mconfig_manager.load_mconfig()
Nov 27 08:29:30 magma-dev magmad[1019599]:   File "/home/vagrant/magma/orc8r/gateway/python/magma/configuration/mconfig_managers.py", line 179, in load_mconfig
Nov 27 08:29:30 magma-dev magmad[1019599]:     raise LoadConfigError('Error loading mconfig') from e
Nov 27 08:29:30 magma-dev magmad[1019599]: magma.configuration.exceptions.LoadConfigError: Error loading mconfig
Nov 27 08:29:30 magma-dev systemd[1]: magma@magmad.service: Main process exited, code=exited, status=1/FAILURE
vagrant@magma-dev:~/magma/lte/gateway$ sudo tail  /var/log/syslog
google.protobuf.json_format.ParseError: Can not find message descriptor by type_url: type.googleapis.com/magma.mconfig.DnsD at GatewayConfigs.configsByKey[dnsd]

**Comment by panyogesh** [comment]
Can you please which version of Magma are you using.

**Reply by vivekshrivastava1516** [reply]
Magma 1.8 version using Vagrant I am using.

https://magma.github.io/magma/docs/basics/quick_start_guide
